CHANGELOG.md

```diff
@@ -12,6 +12,10 @@ Updated SRA-Tools to 2.10.8, which improves stability of the downloading and unp
 
 ### `Added`
 
+## 2020-12-10
+
```

Comment:
```suggestion
```

Reply:
This looks good, thanks for updating the Changelog, but we only add the date of the release, not all the individual changes